Added info for Global Azure PASS Rg Münsterland

Frank-Geisler opened the pull request:
Hi!

Hopefully I have done everything the right way. Otherwise please contact me and I will do corrections.

Thank you and have a nice day

Frank

noopman reviewed (approved):
Great entry! Approved! Welcome to Global Azure @Frank-Geisler and @DiHo78!

Frank-Geisler commented:
Hi Magnus!

 

I don’t see our Communitie on the website. Are there any additional steps necessary?

 

Viele Grüße und vielen Dank

 

Frank Geisler

 

--------------------------------------------------------------



 

GDS Business Intelligence GmbH

Frank Geisler MSc in IT

Geschäftführer

MCT MCSE MCP+I 

 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> Microsoft SQL Server MVP

 

Wolfsberger Strasse 7

59348 Lüdinghausen

Tel.: +49.259.120.695.80
Fax: +49.259.120.695.82

 

E-Mail: ***@***.*** ***@***.***> 

Web:  <http://www.gds-business-intelligence.de/> www.gds-business-intelligence.de 

XING-Profil:  <http://www.xing.com/profile/Frank_Geisler> http://www.xing.com/profile/Frank_Geisler

Twitter: @FrankGeisler

 

Diese Nachricht ist ausschließlich für die adressierte Person oder Organisation (Empfänger) bestimmt und könnte vertrauliche und/oder persönliche Informationen enthalten. Allen Empfängern, für die diese Nachricht nicht bestimmt ist, ist es nicht gestattet, diese zu lesen, zu kopieren, erneut zu übertragen oder anderweitig zu verwenden oder Maßnahmen zu ergreifen, die auf dem Inhalt dieser Nachricht basieren. Sollten Sie diese Nachricht irrtümlich erhalten haben, bitten wir Sie, sich mit dem Absender in Verbindung zu setzen und diese Nachricht unverzüglich von Ihrem Computer zu löschen. Vielen Dank für Ihre Kooperation. 

 

This message transmitted is intended solely for the use of the person or entity to which it is addressed (recipient) and may contain confidential and/or privileged material. If you are not the intended recipient, it is strictly prohibited to review, to copy, to retransmit this message or take any action based on this message or any information herein. If you received this in error, please contact the sender and delete this message from any computer. Thank you for your cooperation.

 

 Bitte denken Sie an die Umwelt. Müssen Sie diese Nachricht ausdrucken?

 

Von: Magnus Mårtensson ***@***.***> 
Gesendet: Mittwoch, 16. März 2022 18:06
An: globalazure/communities ***@***.***>
Cc: Frank Geisler ***@***.***>; Mention ***@***.***>
Betreff: Re: [globalazure/communities] Added info for Global Azure PASS Rg Münsterland (PR #29)

 

@noopman approved this pull request.

Great entry! Approved! Welcome to Global Azure @Frank-Geisler <https://github.com/Frank-Geisler>  and @DiHo78 <https://github.com/DiHo78> !

—
Reply to this email directly, view it on GitHub <https://github.com/globalazure/communities/pull/29#pullrequestreview-911991681> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACM5ZYPQQM3TZ767BXCN7VLVAIIIJANCNFSM5QVS2ZPQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> . 
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ACM5ZYNRPCOICGBRGH2AR5LVAIIIJA5CNFSM5QVS2ZP2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOGZN6HAI.gif> Message ID: ***@***.*** ***@***.***> >

noopman commented:
Hello @Frank-Geisler! I hope that this is seen as a straight forward process. If you feel otherwise I would love to get your feedback! What we did was put the information you should need in this repo's top [readme.md](https://github.com/globalazure/communities/blob/main/README.md) file! The community page here, and the Sessionize data for your event's sessions, are the things for you to check off! Let me know if there are additional questions! /Magnus

Frank-Geisler commented:
Hi Magnus!

 

I am not the person with the strong git knowledge 😉 and I think I did everything right but something is still missing because my folder does not show up in the main branch and therefor my community does not show up on the website. What I did:

 

*	Forked the repo
*	Added a new folder for my usergroup
*	Added and edited all the necessary files
*	Staged them, checked them in (in my repo they show up  <https://github.com/PASS-Deutschland-RG-Munsterland/communities/tree/main/2022> communities/2022 at main · PASS-Deutschland-RG-Munsterland/communities (github.com))
*	Created a pull request  <https://github.com/globalazure/communities/compare/main...PASS-Deutschland-RG-Munsterland:main> Comparing globalazure:main...PASS-Deutschland-RG-Munsterland:main · globalazure/communities (github.com)

 

Then there was a message from you that you accepted the pull request and then I closed it. Sorry if I did something wrong and sorry for the inconvenience. What shall I do now (besides learning how git works 😉)?

 

Thank you very much and have a nice day

 

Frank Geisler

 

--------------------------------------------------------------



 

GDS Business Intelligence GmbH

Frank Geisler MSc in IT

Geschäftführer

MCT MCSE MCP+I 

 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> Microsoft SQL Server MVP

 

Wolfsberger Strasse 7

59348 Lüdinghausen

Tel.: +49.259.120.695.80
Fax: +49.259.120.695.82

 

E-Mail: ***@***.*** ***@***.***> 

Web:  <http://www.gds-business-intelligence.de/> www.gds-business-intelligence.de 

XING-Profil:  <http://www.xing.com/profile/Frank_Geisler> http://www.xing.com/profile/Frank_Geisler

Twitter: @FrankGeisler

 

Diese Nachricht ist ausschließlich für die adressierte Person oder Organisation (Empfänger) bestimmt und könnte vertrauliche und/oder persönliche Informationen enthalten. Allen Empfängern, für die diese Nachricht nicht bestimmt ist, ist es nicht gestattet, diese zu lesen, zu kopieren, erneut zu übertragen oder anderweitig zu verwenden oder Maßnahmen zu ergreifen, die auf dem Inhalt dieser Nachricht basieren. Sollten Sie diese Nachricht irrtümlich erhalten haben, bitten wir Sie, sich mit dem Absender in Verbindung zu setzen und diese Nachricht unverzüglich von Ihrem Computer zu löschen. Vielen Dank für Ihre Kooperation. 

 

This message transmitted is intended solely for the use of the person or entity to which it is addressed (recipient) and may contain confidential and/or privileged material. If you are not the intended recipient, it is strictly prohibited to review, to copy, to retransmit this message or take any action based on this message or any information herein. If you received this in error, please contact the sender and delete this message from any computer. Thank you for your cooperation.

 

 Bitte denken Sie an die Umwelt. Müssen Sie diese Nachricht ausdrucken?

 

Von: Magnus Mårtensson ***@***.***> 
Gesendet: Mittwoch, 16. März 2022 21:00
An: globalazure/communities ***@***.***>
Cc: Frank Geisler ***@***.***>; Mention ***@***.***>
Betreff: Re: [globalazure/communities] Added info for Global Azure PASS Rg Münsterland (PR #29)

 

Hello @Frank-Geisler <https://github.com/Frank-Geisler> ! I hope that this is seen as a straight forward process. If you feel otherwise I would love to get your feedback! What we did was put the information you should need in this repo's top readme.md <https://github.com/globalazure/communities/blob/main/README.md>  file! The community page here, and the Sessionize data for your event's sessions, are the things for you to check off! Let me know if there are additional questions! /Magnus

—
Reply to this email directly, view it on GitHub <https://github.com/globalazure/communities/pull/29#issuecomment-1069560200> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACM5ZYN54JAH7HE2LHQYFKLVAI4SZANCNFSM5QVS2ZPQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> . 
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ACM5ZYMWZIXNCGUHOT4YC7DVAI4SZA5CNFSM5QVS2ZP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOH7ADDCA.gif> Message ID: ***@***.*** ***@***.***> >

noopman commented:
This has probably not been merged. That's odd!

noopman commented:
Now I'll try to merge.

noopman commented:
You're right @Frank-Geisler... Not sure what happened there. I think I may have omitted to merge the PR into our main. I reopened it, and then merged it. It looks like we are okay now! This was my bad, not yours. :)

Frank-Geisler commented:
Hi Magnus!


Awesome! So my first pull request worked 😊. Thank you very much for your support

 

Viele Grüße und vielen Dank

 

Frank Geisler

 

--------------------------------------------------------------



 

GDS Business Intelligence GmbH

Frank Geisler MSc in IT

Geschäftführer

MCT MCSE MCP+I 

 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> 

 <https://mvp.microsoft.com/de-de/mvp/Frank%20Geisler-5000958> Microsoft SQL Server MVP

 

Wolfsberger Strasse 7

59348 Lüdinghausen

Tel.: +49.259.120.695.80
Fax: +49.259.120.695.82

 

E-Mail: ***@***.*** ***@***.***> 

Web:  <http://www.gds-business-intelligence.de/> www.gds-business-intelligence.de 

XING-Profil:  <http://www.xing.com/profile/Frank_Geisler> http://www.xing.com/profile/Frank_Geisler

Twitter: @FrankGeisler

 

Diese Nachricht ist ausschließlich für die adressierte Person oder Organisation (Empfänger) bestimmt und könnte vertrauliche und/oder persönliche Informationen enthalten. Allen Empfängern, für die diese Nachricht nicht bestimmt ist, ist es nicht gestattet, diese zu lesen, zu kopieren, erneut zu übertragen oder anderweitig zu verwenden oder Maßnahmen zu ergreifen, die auf dem Inhalt dieser Nachricht basieren. Sollten Sie diese Nachricht irrtümlich erhalten haben, bitten wir Sie, sich mit dem Absender in Verbindung zu setzen und diese Nachricht unverzüglich von Ihrem Computer zu löschen. Vielen Dank für Ihre Kooperation. 

 

This message transmitted is intended solely for the use of the person or entity to which it is addressed (recipient) and may contain confidential and/or privileged material. If you are not the intended recipient, it is strictly prohibited to review, to copy, to retransmit this message or take any action based on this message or any information herein. If you received this in error, please contact the sender and delete this message from any computer. Thank you for your cooperation.

 

 Bitte denken Sie an die Umwelt. Müssen Sie diese Nachricht ausdrucken?

 

Von: Magnus Mårtensson ***@***.***> 
Gesendet: Donnerstag, 17. März 2022 10:14
An: globalazure/communities ***@***.***>
Cc: Frank Geisler ***@***.***>; Mention ***@***.***>
Betreff: Re: [globalazure/communities] Added info for Global Azure PASS Rg Münsterland (PR #29)

 

You're right @Frank-Geisler <https://github.com/Frank-Geisler> ... Not sure what happened there. I think I may have omitted to merge the PR into our main. I reopened it, and then merged it. It looks like we are okay now! This was my bad, not yours. :)

—
Reply to this email directly, view it on GitHub <https://github.com/globalazure/communities/pull/29#issuecomment-1070592145> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACM5ZYMBOQEDHDEK4UJLBOLVALZURANCNFSM5QVS2ZPQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> . 
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/ACM5ZYPZ524UAS4GCEUJF7LVALZURA5CNFSM5QVS2ZP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOH7H7BEI.gif> Message ID: ***@***.*** ***@***.***> >
